src/Handler/DispatchHandler.php

```diff
@@ -60,7 +60,12 @@ protected function dispatcher()
         return FastRoute\simpleDispatcher(function (RouteCollector $collector) {
             foreach ($this->directory as $request => $action) {
                 list($method, $path) = explode(' ', $request, 2);
-                $collector->addRoute($method, $path, $action);
+
+                $collector->addRoute(
+                    $method,
+                    $this->directory->prefix($path),
```

Comment:
I think prefixing each dispatched URI makes more sense. To me it makes intuitive sense that the request *does* include the prefix, so the route it matches should include that.

Plus I think the code is simpler this way.